### PR TITLE
Fixup internal docs

### PR DESCRIPTION
**Pull request by Niki4tap**
Not sure if that works here, but
r? @xFrednet

### CODE REVIEW

**Review by xFrednet — COMMENTED** [review]
Hey, Marker is currently a separate project and doesn't have access to rust-lang's bots. Luckily, this project has something like `@bors` to test the project on different systems on before merging. I'm subscribed to this repo, so you can generally just comment or ping me directly. :)

Guess your comments show that I didn't have anyone proofreading the documentation. Thank you for the PR, I highly appreciate it. One small question and then I'm happy to merge this :)

**Comment by xFrednet** [comment]
The project currently has no real contributor documentation. Adding that has been on my todo list. However, I wanted to first finish the bare bones of the project before starting on that. You're welcome to ask questions if you're interested or want to help out.

I have exams in the next two weeks, and will be a bit less active, like the last two weeks. But then I plan to come back with full force. My goal is to have an alpha version of 0.0.1 by the end of March.

Have a great start into the next year! :tada:

**Comment by Niki4tap** [comment]
> The project currently has no real contributor documentation.

Yep, I'm aware that the project is very early in development, just helping out wherever I can. I don't think I will be able to help a lot, but improving the docs is the least I can do.

> You're welcome to ask questions if you're interested or want to help out.

Sure, I don't think I will be able to do much in the actual implementation code, but if I do you'll probably see me here :)

> Have a great start into the next year! :tada:

Thanks, you too! :tada:

**Comment by xFrednet** [comment]
I can find some good first issues if you want ^^. Writing and proofreading the documentation is already a big help. With v0.0.1 I'll need to write a bunch of documentation, having someone double check that would be awesome <3

Have you seen the review comment above? :upside_down_face:

**Comment by Niki4tap** [comment]
> I can find some good first issues if you want

That would be great!

> having someone double check that would be awesome

Let's just say that I was interested in the project and noticed some typos in the doc while reading about it :)
But honestly, I'm looking through the code a bit, and noticing some little things in doc comments that can be improved, but I won't PR each one separately, so it might take some time for me to go through the rest of it.

> Have you seen the review comment above?

Yeah, sorry, I left a comment myself and apparently, because of how github handles multiple reviews, it didn't end up responding to your comment, but making another one on my review. Anyways, basically I responded with this:
> That's a good point actually, I don't think there is any apparent advantage over inlining. I'm just used to split the lines when writing markdown, the source looks better and I personally think it looks clearer for people writing/editing the docs later

**Comment by xFrednet** [comment]
> That would be great!

Awesome <3, I have two in mind. I'll write them up after my exams this week.

> That's a good point actually, I don't think there is any apparent advantage over inlining. I'm just used to split the lines when writing markdown, the source looks better and I personally think it looks clearer for people writing/editing the docs later

Okay, would you mind unifying the file that is uses one convention? Then we

**Comment by Niki4tap** [comment]
> Okay, would you mind unifying the file that is uses one convention? Then this will be ready for merge :)

Sure, I tried to split up lines a bit more, check if that's alright

File: docs/internal/README.md [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
-2. `cargo-marker` compiles all defined lint crates and moves the binaries to a single target directory (currently `target/marker/lints`)
+1. `cargo-marker` starts off by verifying that a driver is available
+(In dev mode it will compile the driver for every run)
+2. `cargo-marker` compiles all defined lint crates
+and moves the binaries to a single target directory (currently `target/marker/lints`)
 3. `cargo-marker` sets environment values required by `marker_adapter`
-4. `cargo-marker` starts the driver. Currently, for rustc, this is done by setting `RUSTC_WORKSPACE_WRAPPER` and then calling `cargo check` which will in turn execute the driver like it was the usual rustc binary.
+4. `cargo-marker` starts the driver.
+Currently, for rustc, this is done by setting `RUSTC_WORKSPACE_WRAPPER`
+and then executing `cargo check` which will in turn execute the driver like it was the usual rustc binary.
```
Comment (xFrednet) [inline docs/internal/README.md:29]: TIL, that lines following an item don't need to be indented o.O. I thought they needed to. Would you mind indenting them anyways, if they belong to an indented item?

Sorry, if this is too pedantic, this is the last thing :sweat_smile:  

```suggestion
4. `cargo-marker` starts the driver.
	Currently, for rustc, this is done by setting `RUSTC_WORKSPACE_WRAPPER`
	and then executing `cargo check` which will in turn execute the driver like it was the usual rustc binary.
```

**Comment by xFrednet** [comment]
You can merge this PR afterwards, with `bors r+`

bors delegate+

**Comment by Niki4tap** [comment]
Thanks for the review!

**Comment by xFrednet** [comment]
You're very welcome, the final version looks perfect!

**Pull request merged**